src/components/AirdropCarousel.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import { Dimensions, ViewStyle, View, StyleSheet } from 'react-native';
+import Carousel from 'react-native-snap-carousel';
+
+import { AirdropCarouselCardData } from 'app/consts';
+
+import { AirdropBalanceCard } from './AirdropBalanceCard';
+
+interface Props {
+  items: AirdropCarouselCardData[];
+  styles?: ViewStyle;
+  setRef?: (carouselRef: Carousel<AirdropCarouselCardData>) => void;
+  onItemSnap: (index: number) => void;
+}
+
+const SCREEN_WIDTH = Dimensions.get('window').width;
```

Comment:
we could move it to const, something like 
` export const dimensions = Dimensions.get('window'); `

Reply:
done

Reply:
done